editor/js/editor.js

```diff
@@ -38,7 +40,8 @@
 
     function setContent(propertyName, editorName) {
       if (tabby.editors[editorName].editor) {
-        editorContents[propertyName] = tabby.editors[editorName].editor.getValue();
+        editorContents[propertyName] =
+          tabby.editors[editorName].editor.getValue();
```

Comment:
This is just formatting, right? (Guess we need a lint workflow. 🙂)

Reply:
Yeah, this is just a formatting change -- I'm surprised that this wasn't caught during the initial Prettier formatting!